### ISSUE TITLE
Extractor result feedback information

### ISSUE THREAD

**Issue by FyZoey** [clowder-framework/clowder2#994]
**Key problem to be solved:**
When we don’t have a result, we tell users that

**Suggestions:**
Add one line between the “3. Extractor Result” and the “Restart” button, and say something like, “Sorry there’s a bug in the last step, please go back and fix it to generate the result”, or “This feature is under development, you won’t receive a result yet” (this depends on the reason of not receiving the result, please feel free to change the wording, but the idea is to tell users what happened)

**Comment by lmarini** [comment]
I don't believe we currently show results at all. We should definitively make sure we implement that and also take care of errors as described here.
